readme.md

```diff
@@ -294,6 +299,19 @@ and view the lines right below `{{#*inline "freq"}}`.
 
 </details>
 
+<details>
+<summary><b>Ignoring Frequency Values</b></summary>
+
+- By default, any frequency value containing `❌` is ignored as it represents a value that does not appear in JPDB.
+  If you want to ignore other values, edit the `opt-ignored-freq-value-regex` variable and join the ignored symbols with `|`.
+  For example, to also ignore entries containing a `⚠` symbol, do the following:
+
+  ```handlebars
+  {{~#set 'opt-ignored-freq-value-regex'~}} .*(❌|⚠).* {{~/set~}}
```

Comment:
Similarly to the above, this should be ❌|⚠

The documentation is good, and the only thing I'd add is the case where you don't want to ignore any characters,
which should be the following:
```
{{~#set 'opt-ignored-freq-value-regex'~}} {{~/set~}}
```
Technically, I believe the `regexMatch` call will match an empty string, but the if condition checks that if it the result match is an empty string, so it'll still work regardless For sanity, we could use `$.` instead of the empty string, which should be an impossible-to-match regex. If we don't go with `$.`, there should be a comment about it around the if condition.